agent/src/util/device_plugin_service.rs

```diff
@@ -73,8 +78,6 @@ pub type InstanceMap = Arc<RwLock<HashMap<String, InstanceInfo>>>;
 pub struct DevicePluginService {
```

Comment:
I am not sure what the final implementation will look like but it feels strange to still keep both `instance_name` and `config_name` in the shared `DevicePluginService` struct. Should we have one `name` field and move the other to the `DevicePluginBehavior` struct? We can also leave as is for now